### ISSUE TITLE
Consuming RESTApi

### ISSUE THREAD

**Issue by Kaffiend** [duplicati/duplicati#3299]
Is it possible to call the RESTApi of the web server for current backup status details like the UI does already? I would like to integrate backup status and minimal control into a desktop application of which we are considering Duplicati to facilitate backing up its data and associated images to a remote site for our customers. I imagine a proper Authentication Header would be all that is required..but i dont want to assume anything before i dedicate substantial time to the feature. If we can integrate and brand like this I'd love to sponsor Duplicati

**Comment by Pectojin** [comment]
Yeah, it's definitely possible to use the rest API with 3rd party. 

Sadly I don't think there's much documentation on how to use API.

I've built a python tool that interacts with the API the way you're describing. Maybe it's helpful to look at https://github.com/Pectojin/duplicati-client
